easy_pjax/__init__.py

```diff
@@ -14,7 +14,12 @@
 try:
     from django.template import add_to_builtins
 except ImportError:
-    # import path changed in 1.8
-    from django.template.base import add_to_builtins
+    try:
+        # import path changed in 1.8
+        from django.template.base import add_to_builtins
+    except ImportError:
+        # No more add_to_builtins in 1.9+
+        pass
 
-add_to_builtins("easy_pjax.templatetags.pjax_tags")
+if "add_to_builtins" in vars():
```

Comment:
And get
```
    from django.template.base import add_to_builtins
ImportError: cannot import name 'add_to_builtins'
```
?

The `add_to_builtins()` function is **gone** in django 1.9 – adding easy_pjax to `TEMPLATES[BACKEND][OPTIONS][builtins]` doesn't make it reappear.